File: 백준/Bronze/1076. 저항/저항.java
import java.math.BigInteger;
import java.util.*;

class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        Map<String, String> map = new HashMap<>();
        map.put("black", "0");
        map.put("brown", "1");
        map.put("red", "2");
        map.put("orange", "3");
        map.put("yellow", "4");
        map.put("green", "5");
        map.put("blue", "6");
        map.put("violet", "7");
        map.put("grey", "8");
        map.put("white", "9");
        String s1 = sc.nextLine();
        String s2 = sc.nextLine();
        String s3 = sc.nextLine();

        String n1 = map.get(s1);
        String n2 = map.get(s2);
        int n = Integer.parseInt(n1+n2);
        BigInteger n3 = BigInteger.valueOf((long) Math.pow(10, Integer.parseInt(map.get(s3))));
        System.out.println(n3.multiply(BigInteger.valueOf(n)));
        sc.close();
    }
}

File: 백준/Bronze/1076. 저항/저항.kt
import kotlin.math.pow

fun main() {
    val map = mapOf("black" to 0, "brown" to 1, "red" to 2,
        "orange" to 3, "yellow" to 4, "green" to 5, "blue" to 6,
        "violet" to 7, "grey" to 8, "white" to 9)
    val list = List(3) { map[readln()] }
    println((list[0].toString() + list[1].toString()).toLong() * 10.0.pow(list[2]!!.toDouble()).toLong())
}